tangelo/toolboxes/molecular_computation/tests/test_psi4.py

```diff
@@ -13,17 +13,19 @@
 # limitations under the License.
 
 import unittest
-
-import numpy as np
+import os
 
 from tangelo import SecondQuantizedMolecule
 from tangelo.helpers.utils import installed_chem_backends
 from tangelo.problem_decomposition.oniom.oniom_problem_decomposition import ONIOMProblemDecomposition
+from tangelo.problem_decomposition.qmmm.qmmm_problem_decomposition import QMMMProblemDecomposition
 from tangelo.problem_decomposition.oniom._helpers.helper_classes import Fragment
 from tangelo.toolboxes.molecular_computation.integral_solver_psi4 import IntegralSolverPsi4
 from tangelo.algorithms.variational import SA_OO_Solver, BuiltInAnsatze, ADAPTSolver, iQCC_solver
 from tangelo.molecule_library import xyz_H4, mol_H4_minao, xyz_H2, mol_H4_sto3g_uhf_a1_frozen
 
+pwd_this_test = os.path.dirname(os.path.abspath(__file__))+"/"
```

Comment:
same comment as in other test file: build your alalalala variables here, try to refactor to avoid copy-pasting if its easy, etc